indent/ruby.vim

```diff
@@ -549,6 +569,14 @@ function! s:AfterIndentKeyword(pline_info)
     " level
     if s:Match(info.plnum, s:end_end_regex)
       let ind = indent('.')
+    elseif s:IsAssignment(info.pline, col)
+      if g:ruby_indent_end_alignment == 'hanging'
+        " hanging indent
+        let ind = col + info.sw - 1
+      else
+        " align with variable
+        let ind = indent(info.plnum) + info.sw
+      endif
```

Comment:
Could you give me an example that this block matches? It's easy to check what gets called for a particular code block, but not so easy to come up with an example code block that this fits.